test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2633,6 +2633,138 @@ def forward(self, input):
         x = torch.randn(1, 2, 3, requires_grad=True)
         self.run_test(EmptyBranchModel(), x)
 
+    @unittest.skip("Enable this once ORT version is updated")
```

Comment:
Have these been tested locally?

Reply:
can't test them locally with ORT does not allow running opset 12 functions. 
But I ran these tests with a python/numpy script of the subgraphs to verify the behavior with without weights/ignore_index/reductions